### ISSUE TITLE
[Bug]: Can't create new connection

### ISSUE THREAD

**Issue by arjanterheegde** [Power-Maverick/DataverseDevTools-VSCode#147]
### What happened?

Can't create a new connection. Already tried a different version of the tool and a uninstall. It worked before..
![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/6838586/1cc6d734-74db-4928-9c5b-edd2d03776a4)

### What version of the tool are you using?

v2.1.10

### What version of Visual Studio Code are you running?

1.85.1 / Mac OS M1

### Relevant log output

```shell
Error running command dvdt.explorer.connections.addConnection: command 'dvdt.explorer.connections.addConnection' not found. This is likely caused by the extension that contributes dvdt.explorer.connections.addConnection.
```

**Comment by jacobbaek** [comment]
I faced same issue.

`Error running command dvdt.explorer.connections.addConnection: command 'dvdt.explorer.connections.addConnection' not found. This is likely caused by the extension that contributes dvdt.explorer.connections.addConnection.`

version : 2.1.10

**Comment by allvar** [comment]
Same here on OSX running on a M1.

Tool version 2.1.10
VS Code version 1.85.1
Node version 18.18, but also tried Node version 14.21 with same result.

**Comment by Power-Maverick** [comment]
Found an issue with a related Azure package. Have logged a bug with that team: https://github.com/Azure/azure-sdk-for-js/issues/28112

**Comment by Power-Maverick** [comment]
This is now resolved in v2.2.0. Please update the tool. Thanks for raising the issue.
